scripts/build_packages.py

```
@@ -0,0 +1,330 @@
+
+    with open(setup_py, 'r', encoding='utf-8') as f:
+        tree = ast.parse(f.read(), filename=str(setup_py))
+
+    all_deps = []
+    extras = {}
+
+    for node in ast.walk(tree):
+        if isinstance(node, ast.Call) and getattr(node.func, 'id', '') == 'setup':
+            for keyword in node.keywords:
+                if (keyword.arg == 'install_requires' and
+                    isinstance(keyword.value, (ast.List, ast.Tuple))):
```

Flake8 (7.2.0)

[error] visually indented line with same indent as next logical line

(E129)